### PR TITLE
Remove enable nsfw

### PR DESCRIPTION
**Pull request by Nutomic**
View only the first commit to see main changes.

### CODE REVIEW

File: migrations/2024-09-12-130204_drop-enable-nsfw/up.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-- if site has enable_nsfw, set a default content warning
+UPDATE
+    site
+SET
+    content_warning = CASE WHEN local_site.enable_nsfw THEN
+        'nsfw'
```
Comment (SleeplessOne1917) [inline migrations/2024-09-12-130204_drop-enable-nsfw/up.sql:6]: Nitpick: this context warning might look better if the NSFW acronym is uppercase.

**Review by SleeplessOne1917 — COMMENTED** [review]
Is the motivation behind this change that `enable_nsfw` doesn't affect whether users can mark their posts/communities as NSFW, and that the field is redundant for determining whether or not to use a consent modal since the presence/absence of the `content_warning` alone is enough to tell?

File: migrations/2024-09-12-130204_drop-enable-nsfw/up.sql (hunk 1 above)
Comment (Nutomic) [reply migrations/2024-09-12-130204_drop-enable-nsfw/up.sql]: Changed

**Review by dessalines — APPROVED** [review]
LGTM

**Pull request merged**